plugins/janus_sip.c

```diff
@@ -206,6 +207,7 @@
 	"call_id" : "<value of SIP Call-ID header for related call>",
 	"result" : {
 		"event" : "ringing",
+		"headers" : "<array of key/value objects, optional>"
```

Comment:
The description (here and in the other occurrences) should be a bit more elaborate: e.g., something like "array of custom headers as key/value objects". Besides, the "optional" should come after a semicolon, not a comma (just as a convention, since it's how it's done in the rest of the docs).